pydy/viz/static/index.html

```diff
@@ -96,13 +89,13 @@
               <span class="glyphicon glyphicon-edit" ></span>
               Show Scene JSON
             </button>
+            <label id="checkbox-play-looped"><input type="checkbox" id="play-looped">Play Looped</label>
             <div id="slider-wrapper">
               <span class="glyphicon glyphicon-time" ></span>
-              <span style="margin-right:1em;" id="time"> 0s</span>
-              <div id="time-slider" class="slider" style="float:right; width:28em;padding-left: 2em;"></div>
+              <span style="margin-right:1em;" id="time"> 0.000 s</span>
```

Comment:
Humans can only detect up to 60 hz (~0.02 s), so I don't think showing a third decimal here is needed. Showing only two digits after the decimal should always be sufficient.

But saying that, we currently animate at whatever framerate that is dictated by the time array that is generated. There is a bigger issue here about whether we want to animate at very slowed down frame rates or not.

Anyways, for now we can just stick to two digits if that is  quick fix.